content/history/directors.md

```diff
@@ -17,3 +17,5 @@ It is <em>not</em> an official record - just an informational page.
 <script type="text/javascript">
```

Comment:
Should the whole HTML code in here be "escaped" as it is rendered as text on github now? Do you want this @bdelacretaz ?

Reply:
Perfect - I only saw it in the Github diff and it was rendered without being interpreted. Thx